Applications/ProductRatings/res/product_ratings_model.ini

```diff
@@ -22,13 +22,13 @@ Type = split
 axis = 3 # split by width
 
 [user_embed]
-input_layers = split
+input_layers = split(0)
```

Comment:
please add a todo here to remove the split layer as we now support multiple inputs.

Reply:
Yeah, done